lib/ui/cookie.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env browser */
+
+// https://stackoverflow.com/questions/10730362/get-cookie-by-name
+export default function getCookie(name) {
```

Comment:
Or you could use regular expressions or [some well tested already existing library](https://github.com/js-cookie/js-cookie).